### PR TITLE
another round of testing

### PR DESCRIPTION
**Pull request by kelvinabrokwa**

### CODE REVIEW

File: debug/index.html [hunk 1]
```diff
@@ -68,6 +69,7 @@
 
     Draw.clear();
     console.log(Draw.get(firstId), '-> undefined');
+    */
```
Comment (mcwhittemore) [inline debug/index.html]: ?

File: src/util.js [hunk 2]
```diff
@@ -105,6 +105,7 @@ module.exports.createButton = function(container, opts, controlClass) {
 
     var el = e.target;
 
+    console.log(el.classList, el.classList.contains('active'));
```
Comment (mcwhittemore) [inline src/util.js]: remove

File: src/draw.js [hunk 3]
```diff
@@ -185,10 +185,7 @@ export default class Draw extends API {
   _handleDrawFinished() {
     this._store.getSelectedIds().forEach(id => this._store.commit(id));
     this._hideDeleteButton();
-    [ this.lineStringCtrl,
-        this.polygonCtrl,
-        this.squareCtrl,
-        this.markerCtrl ].forEach(ctrl => { if (ctrl) ctrl.classList.remove('active'); });
+    DOM.removeClass([ this.lineStringCtrl, this.polygonCtrl, this.squareCtrl, this.markerCtrl ], 'active');
```
Comment (mcwhittemore) [inline src/draw.js]: Do we want to keep `DOM` long term? I'm not sure we do enough dom manipulation to warrant abstractions on `document`.

**Comment by mcwhittemore** [comment]
@kelvinabrokwa I think a bunch of this was covered in our last testing push. Closing for now.

**Comment by kelvinabrokwa** [comment]
@mcwhittemore :+1:

**Pull request closed without merging**